src/gam/gapi/cloudidentity/inboundsso.py

```diff
@@ -76,18 +76,17 @@ def profile_displayname_to_name(displayName, ci=None):
     profiles = gapi.get_all_pages(ci.inboundSamlSsoProfiles(),
                                   'list',
                                   'inboundSamlSsoProfiles',
-                                  filter=_filter,
-                                  )
+                                  filter=_filter)
     matches = []
     for profile in profiles:
         if displayName.lower() == profile.get('displayName', '').lower():
             matches.append(profile)
     if len(matches) == 1:
         return matches[0]['name']
-    elif len(matches) == 0:
-        controlflow.system_error_exit(3, f'No Inbound SSO profile matching the name {displayName}')
+    if len(matches) == 0:
```

Comment:
elif this so we don't need to re-eval if len(matches) == 1 above.

Reply:
If len(matches) == 1 we return; they'll be no re-evaluation

Reply:
fair enough